src/v2/view-builder/internals/BaseForm.js

```diff
@@ -113,37 +113,53 @@ export default Form.extend({
   },
 
   /*
-   * Whether an form error message should be rendered as the default BaseForm error banner or not
+   * Decides wether a form error should be rendered as the default BaseForm error banner or customized by view
    * if this function returns true, BaseForm won't render the error banner after form submit. Instead,
-   * the form needs to override showCustomErrorCallout with the logic to render that error message
+   * the view must override showCustomFormErrorCallout
    */
-  isErrorCalloutCustomized() {
+  isFormErrorCalloutCustomized() {
     return false;
   },
 
   /*
-   * Renders an error message (if any) after form is submitted
-   * This should be overriden if isErrorCalloutCustomized() returns true
+   * Views should override this function to render custom error callouts after any user actions.
+   * This should be overriden if isFormErrorCalloutCustomized() returns true
    */
-  showCustomErrorCallout() {
-    this.$('.o-form-error-container').addClass('o-form-has-errors');
```

Comment:
- Remind me why this line can be removed?
- Sound wired if overriding this method depends on value of another method? Do you think we can combine both?

    - set `showCustomErrorCallout` to be `_.identity` or null`

    - In `FormController.showFormErrors`

```
if isFunction(showCustomFormErrorCallout) then 
  form.showCustomFormErrorCallout();
```

Reply:
you think we can get rid of `isFormErrorCalloutCustomized` ? and pass the logic to identify which message is customize and which are not in `showCustomFormErrorCallout` ?.. I think it make sense if thats what you were suggesting..

Reply:
yes , get rid of `isFormErrorCalloutCustomized` which equaling to the exists of function `showCustomFormErrorCallout`.